addOns/ascanrulesAlpha/src/main/javahelp/org/zaproxy/zap/extension/ascanrulesAlpha/resources/help/contents/ascanalpha.html

```diff
@@ -34,5 +34,11 @@ <H2>NoSQL Injection - MongoDB</H2>
 
 Latest code: <a href="https://github.com/zaproxy/zap-extensions/blob/master/addOns/ascanrulesAlpha/src/main/java/org/zaproxy/zap/extension/ascanrulesAlpha/MongoDbInjectionScanRule.java">MongoDbInjectionScanRule.java</a>
 
+<H2>CORS Header\/Misconfiguration</H2>
```

Comment:
```suggestion
<H2>CORS Header Misconfiguration</H2>
```

Not sure how I missed that before.

Reply:
The scan rule default name (getName) is "CORS Header" but it can be CORS Misconfiguration in the alert name. What should I put in the help file?

Reply:
`CORS Header` would be best I guess.